app/components/verkiezingen/burgemeester-alert.js

```diff
@@ -0,0 +1,90 @@
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task, timeout } from 'ember-concurrency';
+import { queryRecord } from 'frontend-lmb/utils/query-record';
+import {
+  MANDAAT_AANGEWEZEN_BURGEMEESTER_CODE,
+  MANDAAT_BURGEMEESTER_CODE,
+} from 'frontend-lmb/utils/well-known-uris';
+
+export default class VerkiezingenBcsdVoorzitterAlertComponent extends Component {
+  @service store;
+  @service bcsd;
+
+  @tracked errorMessage = '';
+  @tracked lastRecomputeTime = null;
+
+  constructor() {
+    super(...arguments);
+    this.bcsd.forceRecomputeBCSD();
+  }
+
+  get collegeOrgaanInTijd() {
+    return this.args.collegeBestuursorgaanInTijd;
+  }
```

Comment:
Not really necessary imo

Reply:
i sortof like this pattern for documentation purposes, this way you see better that there is an arg